src/report.ts

```diff
@@ -19,7 +19,7 @@ function html(vrtResult: DiffResult) {
         <body class="page-comparison">
             <div class="vrt-wrapper">
                 ${navigation()}
-                ${main(vrtResult)}
+                ${main(data)}
```

Comment:
It looks like we just renamed some variables, which are used once within this function scope. Please revert these changes.

Let's keep scope of this PR only to new diff-tool + new options, coming with it in bin + index.